.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,7 @@ Please check if your PR fulfills the following requirements:
 - [ ] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
 - [ ] Tests for the changes have been added (for bug fixes / features)
 - [ ] Docs have been added / updated (for bug fixes / features)
+- [ ] `make test` has completed successfully
```

Comment:
This probably should have been a separate PR.

Reply:
Reverting and creating a separate PR.